readded everything (#104)
MrLongbottom/P10-Article: sections/appendix/author_category_analysis.tex

\subsubsection*{Author-category model analysis}

> While modeling single metadata is the main focus of this paper, looking at the results of models combining multiple metadata may also bring new observations.

Reviewer: ```suggestion
While modeling single metadata fields is the main focus of this paper, looking at the results of models combining multiple metadata may also bring new observations.
```

> For this purpose, we will be examining the results of the Author-Category \gls{lda}.

Reviewer: ```suggestion
For this purpose, we will be examining the results of the Author-Category model.
```

> In \autoref{tab:_extra_metric_results} the metric results for this model is shown, and in \autoref{tab:all_gibbs_topic_examples}, random samples of topics from the \gls{lda}, author-topic, category-topic, and author-category model can be seen.

Reviewer: ```suggestion
In \autoref{tab:_extra_metric_results} the metric results for this model are shown, and in \autoref{tab:all_gibbs_topic_examples}, random samples of topics from the \gls{lda}, author-topic, category-topic, and author-category model can be seen.
```

> In \autoref{tab:_extra_metric_results} the metric results for this model is shown, and in \autoref{tab:all_gibbs_topic_examples}, random samples of topics from the \gls{lda}, author-topic, category-topic, and author-category model can be seen.

Reviewer 2: ```suggestion
In \autoref{tab:_extra_metric_results} the metric results for this model are shown, and in \autoref{tab:all_gibbs_topic_examples}, random samples of topics from the \gls{lda}, author-topic, category-topic, and author-category model can be seen.
```

For the author-category model, the top words in the topics are mostly semantically incoherent, although the author-topic and category-topic models also have topics that are difficult to understand.

> It seems, to some degree, that the topics are a mix of the top words of the two combined metadata topic distributions, which makes sense since we draw word topics from a multiplication of these.

Reviewer 2: ```suggestion
It seems, to some degree, that the topics are a mix of the top words of the two combined metadata topic distributions, which makes sense since we draw word topics from the multiplication of these.
```
